neat/trees/greenstree.py

```diff
@@ -218,7 +214,7 @@ def _collapseBranchToLeaf(self):
     def _setImpedanceArrays(self):
         val = 1. / self.z_soma
         for node in self.child_nodes:
-            val += 1. / node._collapseBranchToRoot()
+            val = val + 1. / node._collapseBranchToRoot()
```

Comment:
is there a reason for this change? if so, maybe comment on it, bc future developers may just change it back.

Reply:
added a comment

Reply:
hmm, i don't understand the comment. where do we check the shape and datatype?

Reply:
For instance if `val` is a numpy array of `float` and `node._collapseBranchToRoot()` is an array of `complex` the first line gives an error, but the second line doesn't. But it appears that in the latest version of the code the datatypes of the arrays are set more carefully to always match because I'm not getting any errors anymore with the original line, so I just changed it back.